src/lib/core/fiber.h

```diff
@@ -1121,6 +1121,26 @@ fiber_c_invoke(fiber_func f, va_list ap)
 	return f(ap);
 }
 
+/**
+ * Whether debug mode is enabled on not.
+ *
+ * Currenly in debug mode we collect fiber gc allocation backtrace and print
+ * it if leak is found. Otherwise we only print that leak is found.
+ */
+extern bool fiber_debug_enabled;
+
+/**
+ * Whether we should abort if gc leak is found. Used in tests.
```

Comment:
Nit: Let's say what it is equal to by default.

Reply:
Done.